File: Polimorfismo/Ejercicio1/Java/Videojuego.java
class Videojuego {
    private String nombre;
    private String plataforma;
    private int cantidadJugadores;
    public Videojuego(String nombre, String plataforma) {
        this.nombre = nombre;
        this.plataforma = plataforma;
        this.cantidadJugadores = 1;
    }
    public Videojuego(String nombre, String plataforma, int cantidadJugadores) {
        this.nombre = nombre;
        this.plataforma = plataforma;
        this.cantidadJugadores = cantidadJugadores;
    }
    public void mostrar() {
        System.out.println("Videojuego: " + nombre + ", Plataforma: " + plataforma + ", Cantidad de jugadores: " + cantidadJugadores);
    }
    public void agregarJugadores() {
        this.cantidadJugadores += 1;
        System.out.println("Cantidad de jugadores actualizada: " + cantidadJugadores);
    }
    public void agregarJugadores(int cantidad) {
        this.cantidadJugadores += cantidad;
        System.out.println("Cantidad de jugadores actualizada: " + cantidadJugadores);
    }
}

File: Polimorfismo/Ejercicio1/Python/Videojuego.py
class Videojuego:
    def __init__(self, nombre, plataforma, cantidadJugadores=1):
        self.nombre = nombre
        self.plataforma = plataforma
        self.cantidadJugadores = cantidadJugadores
    def mostrar(self):
        print("Videojuego: ", self.nombre, "Plataforma: ", self.plataforma, "Cantidad de jugadores: ", self.cantidadJugadores)
    def agregarJugadores(self, cantidad=1):
        self.cantidadJugadores += cantidad
        print(f"Cantidad de jugadores actualizada: {self.cantidadJugadores}")
juego1 = Videojuego("FIFA 24", "PlayStation", 2)
juego2 = Videojuego("Minecraft", "PC")
juego1.mostrar()
juego2.mostrar()
juego1.agregarJugadores()
juego2.agregarJugadores(3) 